app/views/main.scala.html

```diff
@@ -75,18 +75,7 @@
             document.getElementById("wrap").style.padding = "0 0 35px";
         </script>
 
-        <div id="mini-footer-audit">
-          Project Sidewalk is designed and operated by the
-          <a href="https://makeabilitylab.cs.washington.edu/">Makeability Lab</a> at the
-          <a href="http://www.cs.uw.edu/">University of Washington</a><br>
-          <span id="application-version">
-              Version @VersionTable.currentVersionId() | Last Updated: 
-              <span class = "timestamp date">
-                  @VersionTable.currentVersionTimestamp()
-              </span>
-          </span>
-        </div>
-
+        @footer(true)
```

Comment:
I'm not sure how obvious it would be at first glance that this HTML is in a separate file. Maybe just add a comment here saying the footer code is in `app/views/footer.scala.html`? And add the same comment in the other places where you include the footer.